docs/platform-specific/PlatformSpecificDifferencesGenerator.md

```diff
@@ -46,4 +46,4 @@ For a type that has additions, the additions are listed alongside
 
 <!-- Optional -->
 
-- [Platform Specific Analyzer](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.platformspecificanalyzer)
+- [Platform Specific Analyzer](./platformspecificanalyzer)

```

Comment:
Think it's complaining about the link not working within the repo itself, I think this should still translate to the proper location for the full build too:

```suggestion
- [Platform Specific Analyzer](./PlatformSpecificAnalyzer.md)
```